app/Actions/Review/SaveReviewPhotoAction.php

```diff
@@ -25,7 +25,7 @@ public function execute(SaveReviewPhotoRequest $request): SaveReviewPhotoRespons
             $reviewPhoto = $this->reviewPhotoRepository->getById($id);
         }
         $file = $request->getImg();
-        $newFileName = time() . '.' . $file->extension();
+        $newFileName = time() . '_' . mt_rand() . '.' . $file->extension();
```

Comment:
Why we have mt_rand ?

Reply:
when upload more than one photo, it can be uploaded with same name, and will be overwrited

Reply:
@aLLeXUs But photos upload works in async mode, so time() should be enough. Also would be nice to have separate class like FileNameGenerator to create keys like `time() . '_' . mt_rand() . '.' . $file->extension();`, and accept $file as constructor param